config/environments/local.rb

```diff
@@ -22,6 +22,9 @@
   # Only use best-standards-support built into browsers
   config.action_dispatch.best_standards_support = :builtin
 
+  # Turn off asset pipline information showing in logs 
+  config.assets.logger = false 
```

Comment:
@daryanypl @LucasCioffi As mentioned per slack, setting this to true will turn off requested assets from being printed out in the logs. What are both of your thoughts on this? I'd like to get some perspective.

Reply:
Per conversation with Darya, we're going to leave it in for now.